src/modules/app/components/inner-nav/menu-item.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import Styles from 'modules/app/components/inner-nav/inner-nav.styles'
+
+const MenuItem = p => (
+  <li
+    className={classNames({
+      [Styles['InnerNav__menu-item']]: true,
```

Comment:
When you have something that is _always_ true, set it via the first arg of `classNames`.